Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Detect_Service_README.md

```diff
@@ -1,24 +1,20 @@
-Playbook that looks at what ASM sub-type the alert is and directs it to different pre/post mitigation scans (such as NMAP, SNMP).
+Playbook that checks to see if an existing ASM service ID is available. If it is available it will begin a Xpanse Remediation Confirmation Scan.
```

Comment:
```suggestion
Playbook that checks to see if an existing ASM service ID is available. If it is available, it will begin a Cortex Xpanse Remediation Confirmation Scan.
```